### DISCUSSION TITLE
Maximum Files

### DISCUSSION THREAD

**Discussion by Acewiza** [Security-Onion-Solutions/securityonion#6567] [Unsupported Versions] [answered]
I found this steno parameter after noticing my nsm space grossly under-utilized.  I'm curious why this, as opposed to some more basic disk high-water mark-type thing?  Is my 10% of storage space filled with the default 30,000 files going to extrapolate more-or-less accurately for this type storage?  What happens when I guess wrong?  I assume log files must vary in size somewhat based on a plethora of factors.

Any advice or headroom values in mind for that?  I started by quadrupling it to 120,000 - waiting to see where it stops.  Numbers alone seem to point to a 40% fill rate at that value.  A quarter million might be about right, dunno.

**Comment by TOoSmOotH** [comment, marked as answer]
The limit is for certain file systems. Steno will auto prune based on available disk space. So if you are using XFS then you can set that value to a few million.

**Comment by Acewiza** [comment]
Got it.  The old FS issue is mentioned in the doc.  Guess I just didn't realize this data ended up being so small by default for the older file systems - thanks!
